airflow/example_dags/example_snowflake_operator.py

```diff
@@ -0,0 +1,93 @@
+#
```

Comment:
We already have an example_dag: https://github.com/apache/airflow/blob/master/airflow/providers/snowflake/example_dags/example_snowflake.py

we can just use that one or just update that one.

We also store any provider-specific example_dags in that provider's folder itself

Reply:
@kaxil So I understand better, is the howto supposed to cover only the SnowflakeOperator or also all the other examples contained in the already existing example DAG?
In the issue only the SnowflakeOperator was linked, hence the confusion

Reply:
Covering all the operators in the existing would be ideal. Although if you don't know about other operators you can just do it for SnowflakeOperator.

Reply:
Could do about all of them, would probably rework the PR tomorrow and correct where the CI complained about syntax/empty spaces. I assume we're keeping all three then in the same example DAG, I might change it a bit with the additions I had but I'll try to keep it's general structure (I didn't really understand what was the benefit of using env for all Snowflake relevant metadata in the current structure of the example DAG)

Reply:
Yea feel free to update and change and let me know if you have any questions